docs/BOOTING.md

```diff
@@ -79,9 +79,45 @@ Second stage GRUB is expected to do all the [heavy lifting](../pkg/grub/rootfs.c
 an EVE instance and because it resides in IMGA or IMGB partitions it can easily be upgraded and patched.
 Behavior of this stage of boot process is controlled by a read-only grub.cfg under {IMGA,IMGB}/EFI/BOOT/
 but it can further be tweaked by the grub.cfg overrides on the CONFIG partition. Note that an override
-grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L132) and anyone constructing
+grub.cfg is expected to be a [complete override](../pkg/grub/rootfs.cfg#L143) and anyone constructing
 its content is expected to be familiar with the overall flow of read-only grub.cfg.
 
+Option to use in grub.cfg on the CONFIG partition:
+
+1. Cgroup related options. Please follow the [example](README.md#eve-cgroups) to write
+   these options in the correct variables:
+    1. `dom0_mem` - memory limit for eve cgroup (default is `750M,max:750M`)
```

Comment:
Strictly speaking these are not variable names, but rather values of the corresponding GRUB variables. I think we should spell it out explicitly instead of just "options in the correct variables". Or at least give an example here.

Reply:
I put variable to lines